frontend/src/concepts/modelRegistry/types.ts

```diff
@@ -0,0 +1,111 @@
+export enum RegisteredModelState {
+  LIVE = 'LIVE',
+  ARCHIVED = 'ARCHIVED',
+}
+
+export enum ModelVersionState {
+  LIVE = 'LIVE',
+  ARCHIVED = 'ARCHIVED',
+}
+
+export enum InferenceServiceState {
+  DEPLOYED = 'DEPLOYED',
+  UNDEPLOYED = 'UNDEPLOYED',
+}
+
+export enum ExecutionState {
+  UNKNOWN = 'UNKNOWN',
+  NEW = 'NEW',
+  RUNNING = 'RUNNING',
+  COMPLETE = 'COMPLETE',
+  FAILED = 'FAILED',
+  CACHED = 'CACHED',
+  CANCELED = 'CANCELED',
+}
+
+export enum ModelArtifactState {
+  UNKNOWN = 'UNKNOWN',
+  PENDING = 'PENDING',
+  LIVE = 'LIVE',
+  MARKED_FOR_DELETION = 'MARKED_FOR_DELETION',
+  DELETED = 'DELETED',
+  ABANDONED = 'ABANDONED',
+  REFERENCE = 'REFERENCE',
+}
+
+// TODO: add customProperties
```

Comment:
@lucferbux  can you help here to determine the type of the customProperties? [Swagger link](https://editor.swagger.io/?url=https://raw.githubusercontent.com/opendatahub-io/model-registry/main/api/openapi/model-registry.yaml)